addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/receipt.xml

```diff
@@ -185,58 +187,60 @@
             </div>
         </div>
     </t>
-    <t t-name="point_of_sale.OrderLinesReceipt" owl="1">
-        <t t-foreach="receipt.orderlines" t-as="line" t-key="line.id">
-            <div t-esc="line.product_name_wrapped[0]" />
-            <WrappedProductNameLines line="line" />
-            <t t-if="line.display_discount_policy == 'without_discount' and line.price != line.price_lst">
-                <div class="pos-receipt-left-padding">
-                    <t t-esc="env.utils.formatCurrency(line.price_lst, false)" />
-                    ->
-                    <t t-esc="env.utils.formatCurrency(line.price, false)" />
-                </div>
-            </t>
-            <t t-elif="line.discount !== 0">
-                <div class="pos-receipt-left-padding">
-                    <t t-if="pos.config.iface_tax_included === 'total'">
-                        <t t-esc="env.utils.formatCurrency(line.price_with_tax_before_discount, false)"/>
-                    </t>
-                    <t t-else="">
-                        <t t-esc="env.utils.formatCurrency(line.price, false)"/>
-                    </t>
-                </div>
-            </t>
-            <t t-if="line.discount !== 0">
-                <div class="pos-receipt-left-padding">
-                    Discount: <t t-esc="line.discount" />%
-                </div>
-            </t>
+    <t t-name="point_of_sale.ReceiptOrderline" owl="1">
+        <div t-esc="line.product_name_wrapped[0]" />
+        <WrappedProductNameLines line="line" />
+        <t t-if="line.display_discount_policy == 'without_discount' and line.price != line.price_lst">
             <div class="pos-receipt-left-padding">
                 <t t-esc="Math.round(line.quantity * Math.pow(10, pos.dp['Product Unit of Measure'])) / Math.pow(10, pos.dp['Product Unit of Measure'])"/>
                 <t t-if="!line.is_in_unit" t-esc="line.unit_name" />
-                x
                 <t t-esc="env.utils.formatCurrency(line.price_display_one)" />
                 <div class="price_display pos-receipt-right-align">
                     <span t-esc="env.utils.formatCurrency(line.price_display, false)" />
                     <span t-esc="getTaxLetter(...getOrderlineTaxes(line))" class="tax-letter" /> 
                 </div>
```

Comment:
I meant this change. Lines 195-201.

Reply:
Sorry, I was reviewing without whitespaces. Here's what I see from the diff without whitespaces.

<img width="999" alt="Screenshot 2023-08-07 at 10 45 25" src="https://github.com/odoo/odoo/assets/3245568/51ddeb0d-cc0d-46b9-8212-5a4b084e1e30">